benchmarks/nel/project.yml

```diff
@@ -48,10 +55,13 @@ commands:
     script:
       - "unzip -q -d assets/reddit assets/reddit.zip"
       - "python ./scripts/clean_data.py reddit"
+      - "python ./scripts/clean_data.py mewsli-9"
```

Comment:
I think this should be `mewsli_9`? Got an import related error if I didn't change it.